src/api.v2/helpers/worker/getWorkResults.js

```diff
@@ -42,8 +43,8 @@ const validateTagMatching = async (experimentId, params) => {
   }
 };
 
-const getWorkResults = async (experimentId, ETag) => {
-  logger.log(`Getting worker result for experiment: ${experimentId}, Etag: ${ETag}`);
+const tryFetchFromS3 = async (experimentId, ETag) => {
```

Comment:
as I mentioned in the other PR, these are not `ETag`s but only the file name. We could rename it to `fileId` or something like that.